test/lib/next-test-utils.ts

```diff
@@ -171,7 +172,18 @@ export function renderViaHTTP(
 }
 
 export function findPort() {
-  return getPort()
```

Comment:
```js
return import('get-port').then(mod => mod.default())
```

This should work with the latest `get-port` version

Reply:
Out of the box I mean, using our next.js setup. How did you make it work with existing test setup? You can see our test setup doesn't like native esm like 
https://github.com/vercel/next.js/actions/runs/6126906741/job/16631841436?pr=55153#step:29:1292

Course could dig further and trying to bend / adjust our setup, but honestly doesn't feel worth investment unless we want to switch full native esm module support.

Reply:
Yeah I'm guessing jest needs to be configured https://jestjs.io/docs/ecmascript-modules

Reply:
yes, done it once in other codebases. overall, jest & others + native esm wasn't very pleasant experiences yet, and get-port is native esm only (doesn't provide cjs compat at all) unfortunately.